src/Navigation.js

```diff
@@ -1,131 +1,52 @@
-                                    </Nav>
-                                </Accordion.Collapse>
-                            </Card>
-                        </Accordion>
-                    </Nav.Item>
-                    {!isAuth ?
-                        <>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
-                                </Card.Header>
-                            </Card>
-                        </Nav.Item>
-                    }
-                    {isAuth && <Nav.Item>
-                        <Card>
-                            <Card.Header>
-                                <Accordion.Toggle as={Link} to="/" eventKey="0">Welcome back, {user}!</Accordion.Toggle>
-                            </Card.Header>
-                        </Card>
-                    </Nav.Item>}
+      <div className="topnav">
+               <Navbar collapseOnSelect className="ml-auto" expand="lg">
+                <div className="button-right">
+               <Navbar.Toggle  aria-controls='responsive-navbar-nav' />
+               <Navbar.Collapse id="responsive-navbar-nav">
+               <Nav className="ml-auto">
+                
+                      <Link to="/" className="li">
+                         <li >About Us</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Services</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Membership</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>FAQs</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>Blogs</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>Contact Us</li>
```

Comment:
This can be linked to the `GetInTouch` section which was added through pr #229 
<img width="915" alt="Screen Shot 2021-06-18 at 12 07 50 am" src="https://user-images.githubusercontent.com/29667122/122412798-3afd7f80-cfc9-11eb-8dd3-4be042b45251.png">